app/forms/email_verification_form.rb

```diff
@@ -0,0 +1,29 @@
+class EmailVerificationForm < Form
+  attribute :one_time_password
+
+  # Required for shared partial in the view
+  delegate :email_address, to: :claim
+
+  validate :otp_validate
+
+  before_validation do
+    self.one_time_password = one_time_password.gsub(/\D/, "")
```

Comment:
🤔 do we really want to remove all non-digits? I can understand whitespace, but perhaps this is too forgiving.

Reply:
I'm not sure, I just went with what was already in `OneTimePassowrdCeckable#normalise_one_time_password`